docs/sources/reference/components/pyroscope.scrape.md

```diff
@@ -395,8 +396,10 @@ When the `delta` argument is `false`, the [pprof][] HTTP query will be instantan
 When the `delta` argument is `true`:
 * The [pprof][] HTTP query will run for a certain amount of time.
 * A `seconds` parameter is automatically added to the HTTP request.
-* The `seconds` used will be equal to `scrape_interval - 1`.
-  For example, if `scrape_interval` is `"15s"`, `seconds` will be 14 seconds.
+* The default value for the `seconds` query parameter is `scrape_interval - 1`.
+    If you set `profiling_duration`, then `seconds` is assigned the same value as `profiling_duration`.
+    For example, if you set `scrape_interval` to `"15s"`, then `seconds` defaults to `14s`.
+    If you set `profiling_duration` to `16s`, then `seconds` is set to `16s` regardless of the `scrape_interval` value.
```

Comment:
cc @simonswine @korniltsev does it make sense for the `profiling_duration` to ever be larger than the `scrape_interval` here? Or similar to the Prometheus scrape_timeout it should be always less than the scrape interval?

Reply:
It should always be less. If you try to profile a service which is already running a profile, you'll get an error
```
server returned HTTP status (500) Could not enable CPU profiling: cpu profiling already in use
```
I think we should either add some validation, or default back to `min(scrape_interval - 1, profiling_duration)`

Reply:
I think that makes sense as well, yeah cc @hainenber

Reply:
I've added the validation in the HEAD. Looks A-OK to you guys?